packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -68,8 +69,6 @@ abstract contract Sphinx {
 
     SphinxUtils private sphinxUtils;
 
-    bool private isCollecting;
```

Comment:
We previously used this variable to determine whether to prank or broadcast in the `sphinx` modifier. We now always prank, so this variable isn't necessary anymore